app/src/main/java/org/mozilla/focus/ext/Context.kt

```diff
@@ -17,3 +18,11 @@ val Context.appVersionName: String?
         val packageInfo = packageManager.getPackageInfo(packageName, 0)
         return packageInfo.versionName
     }
+
+val Context.isVoiceViewEnabled: Boolean
+    get() {
+        // In my understanding, isTouchExplorationEnabled should represent VoiceView being enabled,
+        // whereas isEnabled says if the accessibility manager is enabled for other services.
```

Comment:
`More details: if you are strictly interested in whether TalkBack is enabled, use am.isTouchExplorationEnabled(). (If Select to Speak is enabled and TalkBack disabled, am.isEnabled() will still return true.)`

https://stackoverflow.com/a/12362545/582004